src/main/java/com/reusable_server/reusableServer/member/presentation/MemberController.java

```diff
@@ -49,15 +57,19 @@ public ApiResponse<?> findOne(@PathVariable Long id) {
 	@GetMapping
 	public ApiResponse<?> findAll() {
 		List<Member> members = memberService.findAll();
+		MemberListResponse memberListResponse = MemberListResponse.of(members);
 		// TODO: DTO로 변환 필요
-		return ApiResponse.of(members);
+		return ApiResponse.of(memberListResponse);
 	}
 
 	@PutMapping("/{id}")
-	public  ApiResponse<?> updateMember(@PathVariable Long id,
-		@Valid @RequestBody MemberUpdateDto memberUpdateDto) {
-		Member updatedMember = memberService.updateMember(id, memberUpdateDto);
-		return ApiResponse.of(updatedMember);
+	public ApiResponse<?> updateMember(@PathVariable Long id,
+		@Valid @RequestBody MemberUpdateRequest memberUpdateRequest) {
+		MemberUpdateParam memberUpdateParam = MemberUpdateParam.from(memberUpdateRequest);
+		Member updatedMember = memberService.updateMember(id, memberUpdateParam);
```

Comment:
수정할 필요 없으시고, 이런 스타일로도 작성이 가능하다 라고만 인지해주시면 될 것 같습니다.

```suggestion
		MemberUpdateParam memberUpdateParam = MemberUpdateParam.from(id, memberUpdateRequest);
		Member updatedMember = memberService.updateMember(memberUpdateParam);
```